translator/translate/logs/logs_collected/files/collect_list/collect_list_test.go

```diff
@@ -149,13 +149,49 @@ func TestTimestampFormatAll(t *testing.T) {
 			input: `{
 					"collect_list":[
 						{
-							"file_path":"path1",
+							"file_path":"path4",
+                            "timestamp_format": "%b %d %H:%M:%S"
+						}
+					]
+				}`,
+			expected: []interface{}{map[string]interface{}{
+				"file_path":         "path4",
+				"from_beginning":    true,
+				"pipe":              false,
+				"retention_in_days": -1,
+				"timestamp_layout":  "Jan 02 15:04:05",
```

Comment:
Does this work for zero padded and non-zero padded days? `02` indicates that it's zero padded only.